docs/current_docs/quickstart/daggerize.mdx

```diff
@@ -82,6 +82,14 @@ dagger init --sdk=typescript --source=./dagger
 
 This will generate a `dagger.json` module metadata file, initial `dagger/src/index.ts` source code template, `dagger/package.json` and `dagger/tsconfig.json` files, as well as a generated `dagger/sdk` folder for local development.
 </TabItem>
+<TabItem value="PHP">
+
+```shell
+dagger init --sdk=php --source=./dagger
+```
```

Comment:
typically we don't specify `./dagger` in PHP projects, we specify `.` to put dagger at the root of the PHP project

this is the way
```
dagger init --sdk=php .
```

Reply:
This produces the module in `.dagger`. To avoid confusion for new users with it being hidden, we explicitly produce it in `dagger`. Leaving this as is for consistency with the rest of the quickstart.